src/model/backup.ts

```diff
@@ -71,7 +70,7 @@ async function extractSecretsForWallet(wallet: RainbowWallet) {
     // Ignore other wallets PKeys
     if (
       item.username.indexOf(`_${privateKeyKey}`) !== -1 &&
-      allowedPkeysKeys.indexOf(item.username) === -1
+      (!allowedPkeysKeys || allowedPkeysKeys.indexOf(item.username) === -1)
```

Comment:
maybe this is better?
```suggestion
      allowedPkeysKeys?.indexOf(item.username) === -1)
```

Reply:
this will not work because undefined for `allowedPkeysKeys` will return false
I can use `!(allowedPkeysKeys?.indexOf(item.username) > -1)` instead